DEVELOPER.md

```diff
@@ -134,6 +134,42 @@ In this section we'll go into how to trigger a breakpoint both in K8S and Univer
 13. Run goland/vscode debugger with remote target on port from point 12
 14. Enjoy!
 
+## Debugging E2E tests interactively
+
+By using the JetBrains Goland IDE, you could also debug the E2E tests running in Kubernetes interactively. To do so, follow these steps:
+
+1. Prepare the images by running `make build`, `make images` and `make docker/tag`
```

Comment:
these could be added as "before launch" steps in the "run configuration" but it will make sense when we persist the whole run config to the repo (see next comment)